foundations/html_css/project/html-css-foundations-project.md

```diff
@@ -20,16 +20,16 @@ As we did in the Recipe project, go ahead and set up a git repository before beg
 
 ### Warning About Looking at Other People's Finished Project Code Before You have Completed Your Own
 
-All of our projects have a "Student Solutions" section at the bottom for people to post their completed code and live websites for the projects. These are here for you to compare your **completed** code to and see what other routes people have chosen to take with their project code and designs. 
+All of our projects have a "Student Solutions" section at the bottom for people to post their completed code and live websites for the projects. These are here for you to compare your **completed** code to, and see what other routes people have chosen to take with their project code and designs. 
 
 We know it may be tempting to look at the solutions when you get stuck or don't know where to start, but **don't** do it! Here are some reasons why:
 
-- All of the submitted projects are other *student* solutions. This means they are written by other people who are also still learning and may not necessarily be representative of "best practice" or quality code.
-- Looking at solutions robs you of the chance to develop your problem solving and researching skills. In the real world, there will not always be "solutions" for you to look at and you will have to come up with your own.
-- There are many different ways to achieve the same end result, it is not "one size fits all".
-- An Analogy: "Looking at completed code is like learning how to make pizza dough from looking at a baked pizza."  There is a lot more that goes into it than you would have assumed by only seeing the final product and not the process that lead to it.
+- All of the submitted projects are other *student* solutions. This means they are written by other people who are also still learning and may not necessarily be representative of "best practice" or high quality code.
```

Comment:
```suggestion
- All of the submitted projects are other *learner's* solutions. This means they are written by other people who are also still learning and may not necessarily be representative of "best practice" or high quality code.
```

Reply:
Right, thanks for reminding me to use learner not student. Although I think it should be *learners'* - will change it.